diff -Nur cmake-3.5.2-orig/Utilities/KWIML/include/kwiml/abi.h cmake-3.5.2/Utilities/KWIML/include/kwiml/abi.h
--- cmake-3.5.2-orig/Utilities/KWIML/include/kwiml/abi.h	2016-04-15 21:41:28.000000000 +0800
+++ cmake-3.5.2/Utilities/KWIML/include/kwiml/abi.h	2017-02-28 12:25:38.000000000 +0800
@@ -365,6 +365,10 @@
 #elif defined(__alpha) || defined(__alpha__) || defined(_M_ALPHA)
 # define KWIML_ABI_ENDIAN_ID KWIML_ABI_ENDIAN_ID_LITTLE
 
+/* sw_64*/
+#elif defined(__sw_64) || defined(__sw_64__) || defined(_M_SW_64)
+# define KWIML_ABI_ENDIAN_ID KWIML_ABI_ENDIAN_ID_LITTLE
+
 /* Arm */
 #elif defined(__arm__)
 # if !defined(__ARMEB__)
